tests/unit/clients/python/test_io.py

```diff
@@ -143,6 +161,105 @@ def test_input_lines_with_jsonlines_docs_groundtruth():
     assert result[1][1].text == "d"
 
 
+@pytest.mark.parametrize(
+    'size, sampling_rate',
+    [
+        (None, None),
+        (1, None),
+        (None, 0.5),
+    ],
+)
+def test_input_huggingface_datasets_from_path(dataset_configs, size, sampling_rate):
+    result = list(
+        from_huggingface_datasets(
+            dataset_configs['adversarial']['dataset_path'],
+            size=size,
+            name=dataset_configs['adversarial']['name'],
+            sampling_rate=sampling_rate,
+            split=dataset_configs['adversarial']['split'],
+        )
+    )
+
+    if size is not None:
+        assert len(result) == size
+
+    assert isinstance(result[0], Document)
```

Comment:
can u please assert the content of some Documents? It is not clear from the implementation in which shape these `Documents` will come

Reply:
I didn't add content specific test care because the dataset is loaded from hub which could change in time. I've added a new test case with content check which uses CSV file as input.